### PR TITLE
[CELEBORN-666][REFACTOR] Unify exclude and blacklist related configurations and variables

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### What changes were proposed in this pull request?
Unify exclude and blacklist related configurations

### CODE REVIEW

**Comment by AngersZhuuuu** [comment]
ping @waitinfuture @FMX @RexXiong @pan3793

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
The change makes sense to me, better to get approval by @waitinfuture before merging

**Comment by waitinfuture** [comment]
I perfer merge https://github.com/apache/incubator-celeborn/pull/1588 before this PR

**Pull request closed without merging**
